pygmt/helpers/decorators.py

```diff
@@ -180,7 +180,10 @@ def use_alias(**aliases):
     R = bla J = meh
     >>> my_module(region='bla', projection='meh')
     R = bla J = meh
-
+    >>> my_module(region='bla', projection='meh', J="bla")
+    Traceback (most recent call last):
+      ...
+    pygmt.exceptions.GMTInvalidInput: Arguments in short-form (J) and long-form (projection) can't coexist
```

Comment:
```suggestion
    >>> my_module(
    ...    region='bla', projection='meh', J="bla"
    ... )  # doctest: +NORMALIZE_WHITESPACE
    Traceback (most recent call last):
      ...
    pygmt.exceptions.GMTInvalidInput:
        Arguments in short-form (J) and long-form (projection) can't coexist
```

Try using [`# doctest +NORMALIZE_WHITESPACE`](https://docs.python.org/3.8/library/doctest.html#doctest.NORMALIZE_WHITESPACE). Just used it recently for the datetime doctest https://github.com/GenericMappingTools/pygmt/blob/9a4c0a0177dac03e05d219c01105f272c9911655/pygmt/clib/conversion.py#L287-L289